clang/lib/Basic/Targets/BPF.cpp

```diff
@@ -67,6 +67,8 @@ void BPFTargetInfo::getTargetDefines(const LangOptions &Opts,
     Builder.defineMacro("__BPF_FEATURE_SDIV_SMOD");
     Builder.defineMacro("__BPF_FEATURE_GOTOL");
     Builder.defineMacro("__BPF_FEATURE_ST");
+    Builder.defineMacro("__BPF_FEATURE_LOAD_ACQUIRE");
+    Builder.defineMacro("__BPF_FEATURE_STORE_RELEASE");
```

Comment:
I think load_acquire and store_release can be just ONE feature as load_acquire/store_release often pair with each other. But I don't have a good name for that. If no good name, your above two features are fine to me too.

Reply:
> But I don't have a good name for that.

+1, but I just noticed that we need to update `clang/test/Preprocessor/bpf-predefined-macros.c`.  I'll do it in the next push.

Reply:
Okay, thanks!

Reply:
It should be one feature. Two macros make it hard for users to reason about the existence of this feature.

Reply:
OK, let me combine them into `__BPF_FEATURE_LOAD_ACQ_STORE_REL`.